pom.xml

```diff
@@ -3326,7 +3326,7 @@
                         <reuseForks>true</reuseForks>
                         <!-- Fail tests longer than 30 minutes, prevent form random locking tests -->
                         <forkedProcessTimeoutInSeconds>1800</forkedProcessTimeoutInSeconds>
-                        <excludedGroups>unstable</excludedGroups>
```

Comment:
Isn't that declared in a bunch of other pom across the project? should we remove them too?

Reply:
well I'm not entirely sure. 

without this change the command 
```
mvn clean test -Dtest=org.apache.james.linshare.LinshareBlobExportMechanismTest -Dsurefire.failIfNoSpecifiedTests=false -am -pl :apache-james-linshare -Dcheckstyle.skip -Punstable-tests
```

doesn´t execute anything, with the change it does run the corresponding tests. 

but there are tests that are tagged `BasicFeature`, unless their module has a custom surefire config to include these tests, they would be excluded by the change i applied

so we probably need something more involved such as `none() |  BasicFeature`

group settings for junit5 are documented at 
https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html#filtering-by-tags
and if I understand correctly it is to be combined with https://junit.org/junit5/docs/current/user-guide/#running-tests-tag-expressions to produce the proper selection.